reana_client/cli/utils.py

```diff
@@ -160,3 +160,62 @@ def handle_parse_result(self, ctx, opts, args):
 
         return super(NotRequiredIf, self).handle_parse_result(
             ctx, opts, args)
+
+
+def output_user_friendly_logs(workflow_logs, steps):
+    """Output workflow logs in an user-friendly manner.
+
+    :param workflow_logs: Dictionary representing the workflow logs as they
+        are returned from the REST API.
+    :param steps: List of steps to show logs for.
+    """
+    def _leading_mark(indent=1):
+        return '{}>'.format('='*indent)
+    # Colors matching the status colors of REANA-UI
+    status_to_color_mapping = {
+        'created': 'magenta',
+        'running': 'bright_blue',
+        'finished': 'green',
+        'failed': 'red',
+        'stopped': 'yellow',
+        'deleted': 'black',
+        'queued': 'bright_white',
+    }
```

Comment:
This is the best I can do with [Click colors](https://github.com/pallets/click/blob/cd35e790c1e41485c18d3e8b5d0891c0280cc96f/examples/colors/colors.py#L4-L7) to show "as close as possible" to the UI ones. CC @mvidalgarcia

Reply:
It looks like this:
![image](https://user-images.githubusercontent.com/9035606/74517692-e0569e00-4f12-11ea-9d68-8ee93e2802c2.png)
Note: In practice, we only use three states for jobs: `running`, `finished` and `failed`, but having a mapping of states as the UI might come handy if we want to color the output for workflows.
<details><summary>Code used to produce this output here (if we want to try other combinations)</summary>
<p>

```python
import click

status_to_color_mapping = {
    'created': 'magenta',
    'running': 'bright_blue',
    'finished': 'green',
    'failed': 'red',
    'stopped': 'yellow',
    'deleted': 'black',
    'queued': 'bright_white',
}

status_to_message_mapping = {
    'created': 'The job is created.',
    'running': 'The job is running',
    'finished': 'The job has finished',
    'failed': 'The job has failed',
    'stopped': 'The job has been stopped',
    'deleted': 'The job has been deleted',
    'queued': 'The job is queued',
}


for status in status_to_color_mapping:
    click.secho(status_to_message_mapping[status],
                fg=status_to_color_mapping[status])

```

</p>
</details>

Reply:
The `created` status can be perhaps the same as `queued`, i.e. default colour, because they express the same thing -- workflow ready to run, as it were.

Reply:
Also, black (or generally dark grey) background colour is a popular setup, so we cannot force `black` colour for `deleted` status... People wouldn't be able to see it.

So all of `created`, `queued`, `deleted` should be rather displayed in the default colour (i.e no click decoration)

You may want to test on Solarized Dark, Solarized Light...